AAP2033 - added quotes to secret values & other minor changes (#349)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/proc-operator-external-db-hub.adoc

[role="_abstract"]
For users who prefer to deploy {PlatformNameShort} with an external database, they can do so by configuring a secret with instance credentials and connection information, then applying it to their cluster using the `oc create` command.

> By default, the {PlatformName} operator automatically creates and configures a managed PostgreSQL pod in the same namespace as your {PlatformNameShort} deployment. A user may instead choose to use an external database if they prefer to use a dedicated node to ensure dedicated resources or to manually manage backups, upgrades, or performance tweaks.

Reviewer: ```suggestion
By default, the {PlatformName} operator automatically creates and configures a managed PostgreSQL pod in the same namespace as your {PlatformNameShort} deployment. You may instead choose to use an external database if you prefer to use a dedicated node to ensure dedicated resources or to manually manage backups, upgrades, or performance tweaks.
```

Reviewer 2: You might instead choose..
